client/src/layout/components/DomainHeader.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react'
+
+// common
+import { Tabs, Tab } from 'common/components/Tabs'
+import { useDomain } from 'common/providers/DomainProvider'
+
+const DomainHeader: FC = () => {
+  const { updateDomainAddress } = useDomain()
+  const tabs = [
+    {
+      title: 'Gemini II',
```

Comment:
probably we should create some domain config JSON with titles and URLs, for now, let's add a TODO to remove the hardcoded title